scripts/go_deadcode.sh

```diff
@@ -16,6 +16,11 @@ exclusions=(
 	"pkg/graph/generated/generated.go" # gqlgen files
 	"/gen/" # Anything in a /gen/ directory (cedar core and cedar intake, namely)
 
+    # Anything in the ./cmd/ directory generally shouldn't be considered dead code, it's usually
+    # just scripts or one-off commands
+    # NOTE: We need to keep the ./cmd/ directory in the deadcode check, as it's where the main entrypoint is for the rest of the code in ./pkg/
+    "^cmd/.*"
```

Comment:
nit - can this indent to match the levels of the other dirs?

Reply:
7ed62d305